Site/admin/components/Account/EmailPassword.php

```diff
@@ -61,6 +61,11 @@ protected function getAccount()
 		return $this->account;
 	}
 
+	// }}}
+	// {{{ abstract protected function getUiXml()
+
+	abstract protected function getUiXml();
```

Comment:
I'm getting this when trying to email a password on CourseHost:
``` 
( ! ) Fatal error: Class SiteAccountEmailPassword contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (SiteAccountEmailPassword::getUiXml) in /so/packages/site/work-nick/Site/admin/components/Account/EmailPassword.php on line 11
```

Reply:
I think you're probably just supposed to be using the parent `getUiXml()` method

Reply:
example https://berna/course-host-ccme/work-nick/www/admin/Account/EmailPassword?id=22600